src/main/java/frc/robot/subsystems/intakerelease/IntakeRelease.java

```diff
@@ -60,12 +60,13 @@ public void setWants(Wants wants) {
   public void periodic() {
     intakeReleaseIO.updateInputs(inputs, wants);
     logger.processInputs("IntakeRelease", inputs);
+    logger.recordOutput("IntakeRelease/State", state.toString());
 
     switch (state) {
       case DISABLED -> intakeReleaseIO.setPercent(0);
       case INTAKE -> intakeReleaseIO.setPercent(-1.0);
       case RELEASE -> intakeReleaseIO.setPercent(0.6);
-      case HOLD_CUBE -> intakeReleaseIO.setPercent(-0.3);
+      case HOLD_CUBE -> intakeReleaseIO.setPercent(0.0); // -0.3
```

Comment:
Why is this 0?

Reply:
we do not want to intake while holding a cube, as it can pop